sdks/python/findSupportedPython.groovy

```diff
@@ -18,7 +18,7 @@
 
 /* This (groovy-maven-plugin) script finds the supported python interpreter and pip
  * binary in the path. As there is no strict naming convention exists amongst OSes
- * for Python & pip (some call it python2.7, others name it python-2.7),
+ * for Python & pip (some call it python3.5, others name it python-3.5),
```

Comment:
This file only seems to be using Python 2, so I wasn't sure if we still needed it.